packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -144,15 +144,6 @@ async function getInfoForAllTeams({ ctx, input }: GetOptions) {
     .findMany({
       where: {
         userId: ctx.user.id,
-        ...(searchString
-          ? {
-              OR: [
-                { user: { username: { contains: searchString } } },
-                { user: { name: { contains: searchString } } },
-                { user: { email: { contains: searchString } } },
-              ],
-            }
-          : {}),
       },
```

Comment:
This is where we fetch team ids where the user belongs to, but we don't have to filter teams with the search string. It's wrong. The filtering should happen later when fetching members from these teams. We already have the logic there, meaning we have duplicated filtering here.